lib/core-integration/src/Test/Integration/Scenario/API/Transactions.hs

```diff
@@ -560,7 +575,7 @@ spec = do
         expectResponseCode @IO HTTP.status404 r
         expectErrorMessage (errMsg404NoWallet $ w ^. walletId) r
 
-    describe "TRANS_CREATE_08 - v2/wallets/{id}/transactions - Methods Not Allowed" $ do
+    describe "TRANS_CREATE_08, TRANS_LIST_04 - v2/wallets/{id}/transactions - Methods Not Allowed" $ do
```

Comment:
This line is quite a bit longer than the limit in our coding standards:

https://github.com/input-output-hk/cardano-wallet/wiki/Coding-Standards#limit-line-length-to-80-characters

There's a strategy for fixing this here:

https://github.com/input-output-hk/cardano-wallet/wiki/Coding-Standards#strategy-3-break-up-long-string-literals

Suggestion: insert a break between the title and the description, like so:
```suggestion
    describe
        "TRANS_CREATE_08, TRANS_LIST_04 - \
        \v2/wallets/{id}/transactions - Methods Not Allowed" $ do
```